plugin/src/test/java/org/opensearch/ml/rest/RestMLDeleteModelActionIT.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.rest;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.apache.http.HttpEntity;
+import org.opensearch.client.Response;
+import org.opensearch.client.ResponseException;
+import org.opensearch.ml.utils.TestHelper;
+import org.opensearch.rest.RestStatus;
+
+public class RestMLDeleteModelActionIT extends MLCommonsRestTestCase {
+
+    public void testDeleteModelAPI_EmptyResources() throws Exception {
+        TestHelper
```

Comment:
To keep consistent with other code, how about using `ExpectedException` ?
```
    @Rule
    public ExpectedException exceptionRule = ExpectedException.none();

    public void testGetModelAPI_EmptyResources() throws IOException {
        exceptionRule.expect(ResponseException.class);
        exceptionRule.expectMessage("index_not_found_exception");
        TestHelper.makeRequest(client(), "DELETE", "/_plugins/_ml/models/111222333", null, "", null);
    }
```

Reply:
Yep, updated to use ExpectedException.